dht.go

```diff
@@ -259,6 +264,9 @@ func makeRoutingTable(h host.Host, cfg config) (*kb.RoutingTable, error) {
 			rtPvLogger.Errorf("failed to connect to peer %s for validation, err=%s", p, err)
 			return false
 		}
+		if !cfg.routingTable.peerFilter(h, h.Network().ConnsToPeer(p)) {
```

Comment:
@aschmahmann : is using the RT peerFilter within the `peerValidationFunction` of the routing table the right place to put it after the kbucket refactor?